velox/substrait/VeloxToSubstraitExpr.h

```diff
@@ -96,6 +97,19 @@ class VeloxToSubstraitExprConvertor {
 
   VeloxToSubstraitTypeConvertorPtr typeConvertor_;
 
+  /// Convert values in Velox flat vector to Substrait List Literal.
+  template <TypeKind flatKind>
+  void flatVectorToListLiteral(
```

Comment:
This method is used only in the .cpp file. Hence, can be a free function in  the anonymous namespace in .cpp file itself.